test/fixtures/sidam.ts

```diff
@@ -76,7 +75,7 @@ async function createUser(ccdCase) {
 }
 
 async function deleteUser(sidamUser) {
-  const email = querystring.stringify(sidamUser.email);
+  const email = encodeURIComponent(sidamUser.email);
```

Comment:
Delete request should be in this form 
curl -X DELETE "https://idam-api.aat.platform.hmcts.net/testing-support/accounts/test9482086%40hmcts.net" -H "accept: */*"